build.gradle

```diff
@@ -85,7 +208,7 @@ task testPython(type:Test) {
     exec {
       environment 'CLASSPATH', sourceSets.main.runtimeClasspath.getAsPath()
       executable = 'python3'
-      args = ['-m', 'pytest', '--junitxml', "$buildDir/test-results/testPython/testPython.xml", '--rootdir', 'python/test']
+      args = ['-m', 'pytest', '-v', '--junitxml', "$buildDir/test-results/testPython/testPython.xml", '--rootdir', 'python/test']
```

Comment:
@eddieh-xlnx - I left in the `-v` by accident, but perhaps it would be useful to keep it in?